Whiskey/Functions/Install-WhiskeyPowerShellModule.ps1

```diff
@@ -27,53 +27,66 @@ function Install-WhiskeyPowerShellModule
     [CmdletBinding()]
     param(
         [Parameter(Mandatory=$true)]
-        [string]
         # The name of the module to install.
-        $Name,
+        [string]$Name,
 
-        [string]
         # The version of the module to install.
-        $Version,
+        [string]$Version,
 
-        [string]
-        # Modules are saved into a PSModules directory. The "Path" parameter is the path where this PSModules directory should be, *not* the path to the PSModules directory itself, i.e. this is the path to the "PSModules" directory's parent directory.
-        $Path = (Get-Location).ProviderPath
+        [Parameter(Mandatory)]
+        # Modules are saved into a PSModules directory. This is the directory where PSModules directory should created, *not* the path to the PSModules directory itself, i.e. this is the path to the "PSModules" directory's parent directory.
+        [string]$BuildRoot,

```

Comment:
This is a decision made in the early days where I didn't want to be passing the context object around everywhere. I thought it would make things easier to test and understand.

It's the build root so that we decrease the knowledge that we actually install things into a PSModules directory from leaking outside as few functions as possible.